tachyon/zk/lookup/halo2/prover_impl.h

```diff
@@ -193,6 +193,9 @@ void Prover<Poly, Evals>::Evaluate(ProverBase<PCS>* prover,
 #define EVALUATE(polynomial, point) \
   prover->EvaluateAndWriteToProof(polynomial.poly(), point_set.point)
 
+  // NOTE(chokobole): The ORDER IS IMPORTANT!!!
```

Comment:
```suggestion
  // The ORDER IS IMPORTANT!!!
```
I thought we were removing NOTE(author) comments. https://github.com/kroma-network/tachyon/commit/98db4aa004b6202d9ca0660b74456f30f1dc2102